utils/fileutils.go

```diff
@@ -34,3 +34,21 @@ func IsDirEmpty(dir string) (bool, error) {
 	}
 	return false, nil
 }
+
+// CopyFile copys src file to dest file
+func CopyFile(src, dst string, perm os.FileMode) error {
```

Comment:
In UNIX world, we usually put dst in front of src 😅

Reply:
👍 😆 Good advice, updated